samples/test/translate_dataset_management_test.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright 2019 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const {assert} = require('chai');
+const {AutoMlClient} = require('@google-cloud/automl');
+const {Storage} = require('@google-cloud/storage');
+
+const cp = require('child_process');
+const uuid = require('uuid');
+
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+
+const CREATE_DATASET_REGION_TAG = 'translate_create_dataset';
+const IMPORT_DATASET_REGION_TAG = 'import_dataset';
+const DELETE_DATASET_REGION_TAG = 'delete_dataset';
+const LIST_DATASET_REGION_TAG = 'list_datasets';
+const GET_DATASET_REGION_TAG = 'get_dataset';
+const DATASET_ID = 'TRL8522556519449886720';
+const EXPORT_DATASET_REGION_TAG = 'export_dataset';
+
+describe('Automl Translate Dataset Tests', () => {
+  const client = new AutoMlClient();
+
+  it('should create, import, and delete a dataset', async () => {
+    const projectId = await client.getProjectId();
+    const displayName = `test_${uuid
+      .v4()
+      .replace(/-/g, '_')
+      .substring(0, 26)}`;
+
+    // create
+    const create_output = execSync(
+      `node ${CREATE_DATASET_REGION_TAG}.js ${projectId} ${displayName}`
+    );
+    assert.match(create_output, /Dataset id:/);
+
+    const datasetId = create_output.split('Dataset id: ')[1].split('\n')[0];
+
+    // import'
+    const data = `gs://${projectId}-automl-translate/en-ja-short.csv`;
+    const import_output = execSync(
+      `node ${IMPORT_DATASET_REGION_TAG}.js ${projectId} ${datasetId} ${data}`
+    );
+    assert.match(import_output, /Dataset imported/);
+
+    // delete
+    const delete_output = execSync(
+      `node ${DELETE_DATASET_REGION_TAG}.js ${projectId} ${datasetId}`
+    );
+    assert.match(delete_output, /Dataset deleted/);
+  });
+
+  it('should list datasets', async () => {
+    const projectId = await client.getProjectId();
+    const list_output = execSync(
+      `node ${LIST_DATASET_REGION_TAG}.js ${projectId}`
+    );
+
+    assert.match(list_output, /Dataset id/);
+  });
+
+  it('should get a dataset', async () => {
```

Comment:
this test, when split into its own file, will likely need a before block where you create the dataset it has to get, you'll keep this logic, and then the addition of an after block where you get rid of the dataset

Reply:
Due to how long these tests take to run, we have pre-existing datasets and models.